### DISCUSSION TITLE
Adding not included gems.

### DISCUSSION THREAD

**Discussion by anteromontonio** [alshedivat/al-folio#2921] [Q&A]
I think there should be some instructions to include Jekyll plugins that were not originally included in the image.
My particular case is described below, but I found a few users with a similar issue.

1. You want to add a jekyll plugin, in my case: [ jekyll-notion](urhttps://github.com/emoriarty/jekyll-notion).
2. You do the standard steps: add the corresponding gem line con the Gemfile: gem 'jekyll-notion'
3. If you try to run the image included as instructed on the INSTALL.md file, this plugin is not installed
4. You try to build your own image and it does not work, I am obtaining the following error:

`/usr/local/bundle/gems/bundler-2.6.2/lib/bundler/definition.rb:676:in 'Bundler::Definition#materialize': Could not find css_parser-1.19.1, nokogiri-1.17.1-aarch64-linux, csv-3.3.0, sass-embedded-1.82.0-aarch64-linux-gnu, rexml-3.3.9, json-2.9.0, google-protobuf-4.29.1-aarch64-linux, google-protobuf-4.29.1, logger-1.6.2, securerandom-0.4.0, ffi-1.17.0-aarch64-linux-gnu in locally installed gems (Bundler::GemNotFound)`

**Comment by george-gca** [comment]
I agree to that. But currently we are experiencing some issues with some library versions. As soon as these are fixed we can take a look at this.
